mesop/tests/e2e/concurrent_updates_websockets_test.ts

```diff
@@ -0,0 +1,17 @@
+import {testInWebSocketsEnabledOnly} from './e2e_helpers';
+import {expect} from '@playwright/test';
+
+testInWebSocketsEnabledOnly(
```

Comment:
Any other test cases we should check that wouldn't be covered by existing tests?